.github/workflows/pytest-workflows.yml

```diff
@@ -38,11 +38,11 @@ jobs:
         # For every workflow, test it with MiniWDL and Cromwell
         tag: ["${{ fromJson(needs.changes.outputs.workflows) }}"]
         engine: ["miniwdl", "cromwell"]
-        exclude:
-          - tag: "wf_theiacov_illumina_pe"
-            engine: "miniwdl"
-          - tag: "wf_theiacov_illumina_se"
-            engine: "miniwdl"
+        #exclude:

```

Comment:
@kapsakcj I may have missed this, but what is the reason for commenting out this exclude block?

Reply:
I commented them out so that the miniwdl tests for those 2 workflows actually run. They were disabled last July because of failures that we were not able to resolve. but I have fixed them and they are now running successfully

If they fail in the future (and we cannot fix them in reasonable amount of time) we can uncomment them and exclude them from the test matrix

Reply:
Got it! Thanks, Curtis